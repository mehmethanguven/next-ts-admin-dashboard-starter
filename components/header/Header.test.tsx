import React from 'react'
import { render } from '@testing-library/react'
import Header from './index'

jest.mock('next/router', () => ({
  useRouter: jest.fn(),
}))
describe('Header test', () => {
  it('should to match snapshot', () => {
    const { asFragment } = render(<Header title="test" />)
    expect(asFragment()).toMatchSnapshot()
  })
})
