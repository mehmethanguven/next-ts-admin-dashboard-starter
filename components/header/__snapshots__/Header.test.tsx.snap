// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test should to match snapshot 1`] = `
<DocumentFragment>
  <div
    class="sticky top-0 z-20 m-4 flex justify-between gap-1 rounded-lg bg-white p-4 shadow-sm"
  >
    <h2
      class="text-xl font-bold"
      data-testid="dashboard"
    >
      test
    </h2>
    <h2
      class="hidden md:block"
    >
      Hello Mehmethan
    </h2>
    <div
      class="md:hidden"
    >
      <svg
        class="h-6 w-6"
        fill="none"
        stroke="currentColor"
        stroke-width="1.5"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
          stroke-linecap="round"
          stroke-linejoin="round"
        />
      </svg>
    </div>
  </div>
</DocumentFragment>
`;
